crates/languages/src/rust/runnables.scm

```diff
@@ -28,6 +29,44 @@
     (#set! tag rust-test)
 )
 
+; Rust doc test
+(
+    (
+        (line_comment) *
+        (line_comment
+            outer: _
+            doc: (_) @_comment_content
+        ) @start
+        (#match? @_comment_content "```")
+        (line_comment) *
+        (line_comment
+            outer: _
+            doc: (_) @_end_comment_content
+        ) @_end_code_block
+        (#match? @_end_comment_content "```")
+        .
+        (attribute_item) *
+        .
+        [(line_comment) (block_comment)] *
+        .
+        [(function_item
```

Comment:
We should probably support `function_signature_item`s too, as that will allow us to support `extern fn` examples and trait item functions.